src/main/java/xyz/vegaone/sixteen/controller/LikeController.java

```diff
@@ -0,0 +1,49 @@
+package xyz.vegaone.sixteen.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import xyz.vegaone.sixteen.dto.Like;
+import xyz.vegaone.sixteen.service.LikeService;
+
+@RestController
+@RequestMapping(value = "/sixteen/like")
+public class LikeController {
+
+    private LikeService likeService;
+
+    @Autowired
```

Comment:
There is a feature since Spring 4.3 that states: if a component/class has only one constructor then you do not need to put `@Autowired` on it, Spring will know to autowire the fields. 

From now on, if the class has only one constructor, leave the `@Autowired` aside.